upstairs/src/control.rs

```diff
@@ -190,126 +171,8 @@ async fn build_downstairs_job_list(up: &Arc<Upstairs>) -> Vec<WorkSummary> {
     let mut jobs = Vec::new();
     for id in kvec.iter() {
         let job = ds.ds_active.get(id).unwrap();
-
-        let (job_type, num_blocks, deps): (String, usize, Vec<u64>) =
```

Comment:
This all just moved to the io_summarize() and ioop_summary() methods.
The only thing that changed really is converting IOState to a string instead of
making up a whole new struct.